pkg/client/errors/meta.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+type Code string
+
+const (
+	SuccessCompletion                             Code = "00000"
+	CodInternalError                              Code = "XX000"
```

Comment:
What the `Cod` prefix stands for ?

Reply:
Cod is only a prefix that could be useful with editor autocompletion, since everything is in the same package. I think here there is a leftover since some entries are not prefixed.